.azure-pipelines/gpu-tests.yml

```diff
@@ -104,10 +104,10 @@ jobs:
       displayName: 'Testing: benchmarks'
 
     - script: |
-        set -e
```

Comment:
you need to keep this, otherwise, it will pass with another bailing than the last bash

Reply:
Done